cuebot/build_hooks.py

```diff
@@ -0,0 +1,44 @@
+
+import os
+import subprocess
+import sys
+
+from hatchling.builders.hooks.plugin.interface import BuildHookInterface
+from hatchling.plugin import hookimpl
+
+class CustomBuildHook(BuildHookInterface):
```

Comment:
Another point against having proto as a cuebot package, now we have python files at the cuebot root directory, which might be confusing without the proper context.

Reply:
Please document this class